app/code/AstoundDRudenko/PriceBadge/etc/di.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <preference for="Magento\ConfigurableProduct\Pricing\Price\LowestPriceOptionsProviderInterface"
```

Comment:
please use same approach for preferences class and namespace as mentioned above.